### DISCUSSION TITLE
Cloning Repositories

### DISCUSSION THREAD

**Discussion by sukanya-pai** [community/community#23062] [New to GitHub]
<p>What is the protocol to be followed when someone clones a repository without permission and uploads the project as their own in their profile without any changes? I personally think it is wrong and it should have been forked, but I am not sure if it is aligning with the community norms. Could you please help me clarify this?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It depends on the license of the code and applicable copyright law. With Open Source licenses uploading copies is usually fine as long as origin information (author, license, history, etc.) in the repository itself is kept intact, with the exact conditions varying by license.</p>
<p>For a specific answer, read the relevant license(s), and consult a lawyer as necessary. If it’s your code and the license is being violated you can take legal action.</p>

**Comment by Sukritish1299** [comment]
I am trying to clone a repository but one single file is not cloning. It shows 100% but just one file is missing every time. What can be the issue?

**Reply by eskayML** [reply]
What file type is the file

**Comment by Sukritish1299** [comment]
It is a folder containing 5 JSON Source Files
